README.md

```diff
@@ -1 +1,114 @@
-# motorbike_booking_frontend
\ No newline at end of file
+# Motorbike Booking Front-End
```

Comment:
- [ ] Your app is really beautiful.  Please, kindly show this off by adding a screenshot to your project readme 💪